index.html

```diff
@@ -0,0 +1,402 @@
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Team Work Practise</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <div class="row">
+          <div class="header__burger">
+            <img src="img/Icon2.png" alt="Burger" />
+          </div>
+          <div class="header__photo">
+            <img src="img/guy.png" alt="Guy" />
+          </div>
+          <h1 class="header__header">
+            free portfolio PSD template FOR GRAPHIC designers
+          </h1>
+          <p class="header__info">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud
+          </p>
+          <button class="header__button" type="button">Hire me</button>
+          <img
+            class="header__arrow"
+            src="img/Arrow.svg"
+            alt="Arrow"
+            width="21"
+            height="36"
+          />
+        </div>
+        <div class="overlay">
+          <div class="container">
+            <div class="header__menu">
+              <img
+                class="header__menu__close"
+                src="img/Iconbl.png"
+                alt="IconBlue"
+              />
+              <nav class="header__nav">
+                <ul class="header__list">
+                  <li class="header__list__item">
+                    <a class="header__list__link" href="#">About Me</a>
+                  </li>
+                  <li class="header__list__item">
+                    <a class="header__list__link" href="#">Services</a>
+                  </li>
+                  <li class="header__list__item">
+                    <a class="header__list__link" href="#">My Works</a>
+                  </li>
+                  <li class="header__list__item">
+                    <a class="header__list__link" href="#">Clients</a>
+                  </li>
+                </ul>
+              </nav>
+              <div class="header__social">
+                <div class="header__social__imgwrap">
+                  <img src="img/Facebook.png" alt="Facebook" />
+                </div>
+                <div class="header__social__imgwrap">
+                  <img src="img/Twitter.png" alt="Twitter" />
+                </div>
+                <div class="header__social__imgwrap">
+                  <img src="img/Dribble.png" alt="Dribble" />
+                </div>
+                <div class="header__social__imgwrap">
+                  <img src="img/Behance.png" alt="Behance" />
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </header>
+    <div class="container">
+      <section class="section1">
+        <h2 class="section1__title section1_title">About Me</h2>
```

Comment:
Тут можна залишити тільки один клас, елемент **title** блоку **section-about**
```suggestion
        <h2 class="section-about__title">About Me</h2>
```